docs/topics/o-TR-Docs.md

```diff
@@ -2,4 +2,8 @@
 
 The o!TR Docs serve to document anything and everything related to the o!TR project. These docs are written using [Jetbrains Writerside](https://www.jetbrains.com/writerside/).
```

Comment:
capitalize JetBrains? (also in line 75 of `API-Getting-Started.md`)

Reply:
also line 5 of ``Getting-Started.md``if consistency matters

separately, line 11 of ``Getting-Started.md`` says "VS Code" instead of "Visual Studio Code" 🤓